apps/frontend/src/app/PageArtifact/ArtifactEditor.tsx

```diff
@@ -248,28 +249,38 @@ export default function ArtifactEditor({
   const {
     old,
     oldType,
+    listContent
   }: {
     old: ICachedArtifact | undefined
     oldType: 'edit' | 'duplicate' | 'upgrade' | ''
+    listContent: Array<ICachedArtifact> | undefined
   } = useMemo(() => {
     const databaseArtifact =
       dirtyDatabase && artifactIdToEdit && database.arts.get(artifactIdToEdit)
-    if (databaseArtifact) return { old: databaseArtifact, oldType: 'edit' }
-    if (artifact === undefined) return { old: undefined, oldType: '' }
+    if (databaseArtifact) return { old: databaseArtifact, oldType: 'edit', listContent: [] }
+    if (artifact === undefined) return { old: undefined, oldType: '', listContent: [] }
     const { duplicated, upgraded } =
       dirtyDatabase && database.arts.findDups(artifact)
+
     return {
       old: duplicated[0] ?? upgraded[0],
       oldType: duplicated.length !== 0 ? 'duplicate' : 'upgrade',
+      listContent: upgraded
     }
   }, [artifact, artifactIdToEdit, database, dirtyDatabase])
 
+
   const { artifact: cArtifact, errors } = useMemo(() => {
     if (!artifact) return { artifact: undefined, errors: [] as Displayable[] }
     const validated = cachedArtifact(artifact, artifactIdToEdit)
-    if (old) validated.artifact.location = old.location
-    return validated
-  }, [artifact, artifactIdToEdit, old])
+    switch (listContent.length > 1) {
+      case true:
+        return { artifact: validated.artifact, errors: ["You have more than one artifact that can be upgraded, please update it manually."] as Displayable[] }
```

Comment:
Can you translate the error string? You can do so by adding a new key-value pair to `artifact.json`, preferably under the `editor` object.

Then you can get the translated value with `t` function.

For example, in `artifact.json`:
```json
{
  "editor": {
    ...
    "multipleUpgrades": "You have more than ..."
...
```

and then in `ArtifactEditor.tsx`: 
```tsx
return { artifact: validated.artifact, errors: [t("editor.multipleUpgrades")] }
```